doc/pinocchio_cheat_sheet.tex

```diff
@@ -0,0 +1,632 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Pinocchio Cheat Sheet
+%
+% Started by Aurélie Bonnefoy
+% Edited by Fabian Schramm
+%
```

Comment:
It would be cleaner to set @ABonnefoy author of a commit that add the initial file, and then @fabinsch could make changes after, without the need for this header :)